### PR TITLE
Adding new features

### PR DESCRIPTION
**Pull request by georgethegreatat**
- Changings in Values, Data, Output & main script

### CODE REVIEW

File: application.tf [hunk 1]
```diff
@@ -1,12 +1,91 @@
...
+  name: game-2048
+YAML
 
+  wait       = true
   depends_on = [helm_release.aws_ingress]
+}
+
+resource "kubectl_manifest" "game-2048-deployment" {
+  yaml_body =<<YAML
+apiVersion: apps/v1
```
Comment (kopachevsky) [inline application.tf:20]: is it possible to expose yaml to standalone yaml file?

File: aws_alb_ingress_controller.tf [hunk 2]
```diff
@@ -1,10 +1,11 @@
-######################################
-###... EKS manual script jobs
-######################################
+#########################################
+### TargetGroupBinding     Deployment ###
+### AWS Ingress Controller Deployment ###
+#########################################
 
 # Create ServiceAccount AWS ALB controlle and attach it with kubernetes cluster
```
Comment (kopachevsky) [inline aws_alb_ingress_controller.tf:6]:
```suggestion
# Create ServiceAccount AWS ALB controller and attach it with Kubernetes cluster
```

File: application.tf [hunk 3]
```diff
@@ -1,12 +1,91 @@
 ####################################################
-###...Apply The 2048 game in Kubernetes Cluster
-###...to verify AWS ALB ingress controller work
+###    Game-2048 Deployment                      ###
```
Comment (kopachevsky) [inline application.tf:2]: just write this is example deployment

File: eks_cluster.tf [hunk 4]
```diff
@@ -1,5 +1,5 @@
 ###########################################
-###...Creation Kubernetes Cluster
+### Kubernetes Cluster. Step 2          ###
 ###########################################
 
 module "kubcluster" {
```
Comment (kopachevsky) [inline eks_cluster.tf:5]:
```suggestion
module "kubecluster" {
```
shortcut for Kubernete sis kube, not kub

File: variables.tf [hunk 5]
```diff
@@ -1,3 +1,7 @@
+########################
+#### Variable values ###
```
Comment (kopachevsky) [inline variables.tf:2]: variable values does not means anything to anyone reading this, better to do not have any comment than obvious one

File: variables.tf [hunk 6]
```diff
@@ -29,7 +33,7 @@ variable "aws_secret_shared_key" {
 }
 
 variable "kub_user_arn" {
-  default     = ""
+  default     = "arn:aws:iam:::user/"
```
Comment (kopachevsky) [inline variables.tf:36]: roles vairalbe also needed

**Review by kopachevsky — CHANGES_REQUESTED** [review]

File: application.tf (hunk 1 above)
Comment (georgethegreatat) [reply application.tf:20]: Fixed in next PR.

File: aws_alb_ingress_controller.tf (hunk 2 above)
Comment (georgethegreatat) [reply aws_alb_ingress_controller.tf:6]: Fixed in next PR.

Comment (georgethegreatat) [reply aws_alb_ingress_controller.tf:6]: Fixed in next PR.

File: eks_cluster.tf (hunk 4 above)
Comment (georgethegreatat) [reply eks_cluster.tf:5]: Fixed in next PR.

File: variables.tf (hunk 6 above)
Comment (georgethegreatat) [reply variables.tf:36]: Fixed in next PR.

File: variables.tf (hunk 5 above)
Comment (georgethegreatat) [reply variables.tf:2]: Fixed in next PR.

File: application.tf (hunk 3 above)
Comment (georgethegreatat) [reply application.tf:2]: Fixed in next PR.

**Review by kopachevsky — APPROVED** [review]

**Pull request merged**